starter-code/index.js

```diff
@@ -1,10 +1,38 @@
-var SortedList = function() {};
-
-SortedList.prototype.add     = function(item) {}
-SortedList.prototype.get     = function(pos) {}
-SortedList.prototype.max     = function() {}
-SortedList.prototype.min     = function() {}
-SortedList.prototype.average = function() {}
-SortedList.prototype.sum     = function() {}
+var SortedList = function() {
+    this.items = [];
+    this.length = 0;
+};
+
+SortedList.prototype.add = function(item) {
+    this.items.push(item);
+    this.length++;
+    this.items.sort(function(item1,item2){
+        return item1 - item2;
+    })
+}
+
+SortedList.prototype.get = function(pos) {
+    return this.items[pos - 1];
+}
+
+SortedList.prototype.max     = function() {
+    return this.items[this.length - 1];
+}
+
+SortedList.prototype.min     = function() {
+    return this.items[0];
+}
+
+SortedList.prototype.average = function() {
+    return this.items.reduce(function(acc, item , index , array){
+        return index < array.length - 1  ? acc + item : (acc + item)/ array.length
+    },0);
+}
+
+SortedList.prototype.sum  = function() {
+    return this.items.reduce(function(acc, item){
+        return acc + item;
+    },0);
+}
```

Comment:
Igualmente, aunque podríais haber complicado menos el `reduce()`, es mucho más adecuado si reutilizáis el `reduce()` que utilizáis en las funciones `sum()` y `average()`. Es decir, lo hacéis en `sum()` y llamáis a `sum()` desde `average()` con `this.sum()`. Así, si os ha surgido algún problema haciendo el reduce en `average` lo solventáis y os queda mucho más limpio